### DISCUSSION TITLE
[Feature] Album Directories on filesystem

### DISCUSSION THREAD

**Discussion by Donderda** [immich-app/immich#3779] [Feature Request]
### The feature

It would be great if albums were reflected as 'symlink' folders in the file system. This way, you could quickly copy the photos of a party or new family members to a USB stick for friends and family without them needing to download the photos from a server (which could have a slower connection, if in doubt).

Fundamentally, the idea would be not to remove the files from the main directory, but to place them as symlinks in corresponding album folders.

Or is such a feature already available?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Closing as a duplicate of #2933/#1006

**Reply by Donderda** [reply]
Well I’m not talking neither about watch folder features nor importing media in general, but about the handling of images in general :-/.

Or did I miss something?

**Reply by chefkoch** [reply]
I agree. The idea seems to be less related to the import and more related to the export or availability of the pictures and albums outside of immich, while keeping the management of the albums within immich.

Reopening the feature request would be great so the topic might catch interest to be discussed.

**Comment by chefkoch** [comment]
Another use case, and I would be interested how other solve this one, would be :   
1. management of albums whitin immich, 
2. Defining specific albums for availability on the filesystem using symlinks or hardlinks
3. Integrating these folders in other software like dynamic slideshow / digital signage software, home assistant , or even for access on your tv or similar.

**Reply by chefkoch** [reply]
Within immich maybe even dynamic folders or filters could be defined and saved permsntly, i.e. people x and y from years 2021-2023, rating 3+

This way you could access the pictures in a slideshow randomly given some context, without the need to specifically define pictures or albums. This could be useful for a wall mounted screen showing pictures of guests you are having at home mixed with your personal last holidays.
